.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+<!--
```

Comment:
There are things that I currently have that I don't yet have well-defined enough to target a release cycle but I think it's worth them being shared, e.g. replacing fitting engine, so that people know about them.

The [new templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository) allow multiple types. Should we have a "capability" type and a "future"/"horizon" type?

Reply:
I don't quite see the difference between these types. Naively, I assume that anything that isn't in the current release is "future" or horizon and will be revisited. Since an individual item will be what a contributor (e.g. facility) intends to do during a release cycle, I would expect reshuffling them would be done by the contributor that "owns" it.

Reply:
I guess it's more that we have the note here that it should already be sized to fit in a single cycle but we've yet to scope things like the "fitting engine replacement" but I'd like people to know that it's on our roadmap - is that something that we wouldn't include here until it's clearer what the breakdown of the task is?

Reply:
Thinking "out loud"... For replacing the fitting engine (or my favorite: get rid of poco), I can imagine it going two ways:
1. We'd probably bring it up as a separate discussion during a steering committee meeting (and email) and find out who would want to be involved so we get a better idea of resourcing and decide how to break it down into capabilities that would go into different releases. It may be that we have parallel systems for a time as well, but it is the people doing the work that have the most input into how it happens.
2. Add in an item in a future release and bring up that it is under-scoped and needs broken down. This would promote the same discussion, but it would be written.

Things in the current (or soon to start) release should definitely be written to fit in, but I wouldn't be surprised if capabilities 2 iterations out don't have a "good" estimate on whether they'd fit into a single release cycle.

Reply:
Yeah I agree that upcoming release work should definitely (or highly likely unless something unforeseen happens) fit in or there is no point in targeting that release.

I like the idea of kicking off a discussion about the items to work out available resources but also having something where the discussion is visible and written down. These items could be issues with labels "needs-scoping" and the discussion happens there - that way it's all in one place and visible.

Once we nail this "how to work with roadmap" down a bit we should put the process in the README.